Penteract/Assets/Scripts/CameraController.cpp

```diff
@@ -42,12 +44,30 @@ void CameraController::Update() {
 	if (playerController == nullptr || transform == nullptr) return;
 	float3 playerGlobalPos = playerController->playerFang.playerMainTransform->GetGlobalPosition();
 
-
 	float3 desiredPosition = playerGlobalPos + float3(cameraOffsetX, cameraOffsetY, cameraOffsetZ);
-	float3 smoothedPosition = desiredPosition;
 
+	float3 smoothedPosition = desiredPosition;
+		
 	if (useSmoothCamera) {
-		smoothedPosition = float3::Lerp(transform->GetGlobalPosition(), desiredPosition, smoothCameraSpeed * Time::GetDeltaTime());
+		if (playerController->playerFang.isAiming()) {
+			float2 mousePosition = Input::GetMousePositionNormalized();
+
+			if (mousePosition.x > 0.75) { aimingPositionX = aimingDistance; }
```

Comment:
It would be nice to have this threshold value exposed, but let's do that in further PRs